addons/binding/org.openhab.binding.yamahareceiver/README.md

```diff
@@ -12,11 +12,11 @@ Initially a thing for the main zone will be created. This will trigger a zone de
 When using zones feature, to manually add a receiver, use
 
 ```
-Bridge yamahareceiver:yamahaAV:ReceiverID "Yamaha Receiver Bridge Name" [HOST="a.b.c.d"] {
-	Thing zone ZoneID1 "Main Zone Thing Name" @ "location" [ZONE="Main_Zone"]
-	Thing zone ZoneID2 "Zone 2 Thing Name" @ "location" [ZONE="Zone_2"]
-	Thing zone ZoneID3 "Zone 3 Thing Name" @ "location" [ZONE="Zone_3"]
-	Thing zone ZoneID4 "Zone 4 Thing Name" @ "location" [ZONE="Zone_4"]
+Bridge yamahareceiver:yamahaAV:ReceiverID "Yamaha Receiver Bridge Name" [host="a.b.c.d"] {
```

Comment:
@davidgraeff @martinvw here are the new settings that follow the configuration bean requirements...

Reply:
@kaikreuzer this breaking change is a result of the limitations of the configuration class mapping, do you agree that it is worth it?

Reply:
In OpenHab 1.x times, it was fancy to use uppercase configuration keys. That has changed with ESH and OpenHab 2.x were the majority of all extensions and bindings are using camelcase configuration keys. Should be unified in my opinion.

Reply:
@martinvw yes, the braking change is dictated by the limitation of the config class mapping. Having a feature in ESH that David suggested would really help here. Also looking at ESH it should be easy to add to ESH. 

@davidgraeff are you saying configs should be upper camel case here and that the mapping class would work? If so, then I could change this. Let me know.

Reply:
> @kaikreuzer this breaking change is a result of the limitations of the configuration class mapping, do you agree that it is worth it?

I leave that decision to the authors - any outcome is fine with me.